cmd/helmExecute.go

```diff
@@ -120,3 +129,42 @@ func runHelmExecuteDefault(config helmExecuteOptions, helmExecutor kubernetes.He
 
 	return nil
 }
+
+//getAndRenderImageInfo gets data from commonPipelineEnvironment
+func getAndRenderImageInfo(config helmExecuteOptions, rootPath string, utils kubernetes.DeployUtils) error {
+	cpe := piperenv.CPEMap{}
```

Comment:
since we are not only interested in some of the values but all of them, I guess a mapping via `resourceRef` will not help. In the step `golangBuild` there is a similar implementation.